README.md

```diff
@@ -25,13 +25,18 @@ This project can be run from any type of host with no platform restrictions. How
 
 #### Running
 
-0. You need to install ansible and terraform first
+0. You need to install ansible and terraform first, or use this docker image
+```
+$ docker run -it -v ${PWD}:/tmp stakater/pipeline-tools:1.0 sh
+$ cd /tmp
+```
 
 1. Run these,
 
 ```
 $ mkdir -p build
 $ touch tfvars.sh
+$ touch secrets.sh
```

Comment:
Add a one line description to demonstrate the difference in b/w secrets.sh and tfvars.sh